tools/generate-hash.mjs

```diff
@@ -0,0 +1,91 @@
+import fs from 'fs/promises';
+import hasha from 'hasha';
+import minimatch from 'minimatch';
+import shell from 'shelljs';
+
+const options = {
+  algorithm: 'md5',
+};
+const hashMap = 'export const hashMap = new Map();';
+let hashes = [];
```

Comment:
I think this should be inlined, otherwise you shadow the variable. We don't need this on module scope.